### ISSUE TITLE
[Bug] VshCounter Plugin MOD

### ISSUE THREAD

**Issue by PS3-Pro** [aldostools/webMAN-MOD#698]
Aldo, when loading the plugin on HEN, it won't show the clock speeds as intended, but it will leave a blank space in the place of it.

I reported it to @TheRouletteBoi and he told me that this does not occur in his vanilla plugin, only in the webMAN Modded version.

![image](https://user-images.githubusercontent.com/74815634/166746710-5e415046-6686-4d28-a866-70ce1fff7ff3.png)

**Comment by aldostools** [comment]
Yes, it happens because I had to add a line feed character before the label "Fan Speed" to make all the clock speeds show in a single line.

I suggested the feature to TheRouletteBoi, but I assume he didn't like it (he added a commit for the bug report, but didn't answer about the idea)
https://github.com/TheRouletteBoi/RouLetteVshMenu/issues/15

This only happen on HEN because it cannot access LV1 to show the clock speeds. On CFW it shows fine.
